test/Chainweb/Test/Mempool/RestAPI.hs

```diff
@@ -70,7 +70,7 @@ newTestServer = mask_ $ do
     server inMemCfg inmemMv envMv restore =
         InMem.withInMemoryMempool inMemCfg version $ \inmem -> do
             putMVar inmemMv inmem
-            restore $ withTestAppServer True version (return $! mkApp inmem) mkEnv $ \env -> do
+            restore $ withTestAppServer True version (return mkApp) mkEnv $ \env -> do
```

Comment:
This is likely the cause of the failures.

Reply:
... but why? The actual chainweb lib no longer uses those "generic" mempools anymore...